build-support/ci_blacklists/integration_test_chroot_blacklist.txt

```diff
@@ -0,0 +1,59 @@
+tests/python/pants_test/backend/codegen/antlr/java:integration
```

Comment:
Majority of these failures are due to any JDK generated code failing to be found. Anything that uses Java/Scala generated code will fail with `--chroot`.